src/main/java/org/carlmontrobotics/robotcode2023/Constants.java

```diff
@@ -13,13 +19,98 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+
+    public static final class Roller {
+        //#region Subsystem Constants
+
+        public static final double coneIntakeConeOuttakeSpeed = -.3;
+        public static final double coneOuttakeConeIntakeSpeed = .7;
+
+        public static final int ledLength = 85;
+        public static final double ledDefaultColorRestoreTime = 3; // The time in seconds after picking up a game piece to restore the LED color to defaultColor
+        public static final Color defaultColor = new Color(0, 0, 200);
+        public static final Color pickupSuccessColor = new Color(0, 200, 0);
+        public static final Color conePickupColor = new Color(150, 150, 0);
+        public static final Color cubePickupColor = new Color(50, 0, 200);
+
+        //#endregion
+
+
+        //#region Ports
```

Comment:
What are these "#region ports"? Are they important or can I just remove them?

Reply:
The `#region` `#endregion` tags are used for code folding. They mark "regions" of code that the ide can collapse, hovering over the line number in vs code and a little arrow you can click should appear next to it